composer.json

```diff
@@ -74,17 +74,43 @@
     "sort-packages": true
   },
   "scripts": {
-    "tests:docker": "docker compose run --rm tests",
     "tests": [
       "@tests:phpinsights",
       "@tests:phpstan",
       "@tests:psalm",
       "@tests:pest"
     ],
-    "tests:phpinsights": "@php ./vendor/bin/phpinsights -v --no-interaction",
-    "tests:phpstan": "@php ./vendor/bin/phpstan analyse --ansi --memory-limit 512M",
-    "tests:psalm": "@php ./vendor/bin/psalm",
-    "tests:pest": "@php ./vendor/bin/pest --stop-on-failure --coverage",
-    "tests:infection": "@php ./vendor/bin/infection --threads=4 --test-framework=pest"
+    "tests:phpinsights": [
```

Comment:
- Disable Composer's process timeout for scripts.
- Add new Dockerfile tests for 7.4, 8.0 and 8.1.